core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -1,9 +1,9 @@
 package io.quarkus.runtime.logging;
 
+import static io.smallrye.common.net.HostName.getQualifiedHostName;
+import static io.smallrye.common.os.Process.getProcessName;
 import static java.util.Collections.emptyList;
 import static java.util.Collections.emptyMap;
-import static org.wildfly.common.net.HostName.getQualifiedHostName;
-import static org.wildfly.common.os.Process.getProcessName;
```

Comment:
TBH, this is what made me think about it. Mostly that it could be counter productive having things depending on both implementations.

Reply:
I think this makes sense. The main cost is borne by class loading, so every redundant class we don't load is a win even if we still use a couple of classes from the library in the end.